pages/index.tsx

```diff
@@ -128,17 +128,21 @@ export default class extends React.Component<IProps, IState> {
     let { lastTickHadColon } = this.state
     let time = whatTimeIsIt()
 
-    this.setState({
-      ...time,
-      lastTickHadColon: !lastTickHadColon
-    })
+    this.setState(time)
   }
 
   componentDidMount() {
     this.tick()
     setInterval(() => {
       this.tick()
     }, 1000)
+    
+    // Let colons blink twice a second
+    setInterval(() => {
+      this.setState({
+        lastTickHadColon: !lastTickHadColon
```

Comment:
Here's the issue, `lastTickHadColon` is not in scope. You have to get it from state first inside the `setInterval` callback

Reply:
That happens when using the GitHub editor :P